atest/testdata/standard_libraries/process/process_library.robot

```diff
@@ -41,6 +41,12 @@ Input things to process
     ${result}=    Wait For Process
     Should Match    ${result.stdout}    *inp 42*
 
+Assign process object to variable
+    ${process} =  Start Process  python  -c  print('Hello, world!')
+    ${result} =  Run Process  python  -c  import sys; print(sys.stdin.read().upper().strip())  stdin=${process.stdout}
+    Wait For Process  ${process}
+    Should Be Equal As Strings  ${result.stdout}  HELLO, WORLD!
```

Comment:
Our convention is to use four spaces (or more if needed) between arguments and keywords.

Reply:
fixed